src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -206,7 +333,27 @@ public static void setMythicVariableMap(Entity entity, Map<String, Variable> map
         registry.putAll(map);
     }
 
+    public static ItemExecutor getItemManager() {
+        return MythicBukkit.inst().getItemManager();
+    }
+
+    public static Collection<String> getItemNames() {
+        return getItemManager().getItemNames();
+    }
+
+    public static MythicItem getMythicItem(String name) {
+        return MythicBukkit.inst().getItemManager().getItem(name).orElse(null);
+    }
+
+    public static Collection<String> getSkillNames() {
+        return MythicBukkit.inst().getSkillManager().getSkillNames();
+    }
+
+    public static String getFaction(ActiveMob mob) {
+        return mob.getFaction();
```

Comment:
Is there a reason this is it's own method, as opposed to just using `ActiveMob#getFaction`?